internal/pkg/log/log.go

```diff
@@ -0,0 +1,83 @@
+package log
+
+import (
+	"octavius/internal/pkg/constant"
+	"os"
+	"time"
+
+	"github.com/rs/zerolog"
+)
+
+type engine struct {
+	logger *zerolog.Logger
+	// put configurabe field here
+}
+
+var logEngine engine // contain cli configarution
+
+// Init intializes the logger object
+// TODO: cater logger requirement for cli and control plane
+func Init(configLogLevel string, logFile string) error {
```

Comment:
Instead of passing in a filename, can we do something like passing in a string param like "executor" or "client" and then use a switch case to configure the `multi writer` as that is the one changing across logs.

Reply:
depend on the case, what I saw is the cases are: some binary need to multi write to file and std out, and others only need to std out. and we can switch it over if logFile is provided. this would make the log package more flexible. 

If we pass executor or client, the only difference also one will write to file and the other is no. but the parameter is coupled with the project. this would make log only available to use for executor or client.